openwisp_controller/config/static/config/js/unsaved_changes.js

```diff
@@ -98,5 +98,11 @@
         });
         // bind unload event
         $(window).bind('beforeunload', unsavedChanges);
+        $(document).on("click", "input.previewlink", function () {
```

Comment:
Please keep the quoting stylee consistent (use `'` instead of `"`).

Reply:
Fixed it!